libs/utils/samplerum.js

```diff
@@ -145,7 +146,7 @@ const defer = (rum, fnname) => {
 function sampleRUM(checkpoint, data = {}) {
   try {
     if (!sampleRUM.initialized) { initSampleRUM(sampleRUM, checkpoint, data); }
-    if (!window.hlx.rum) { initHlxRUM(sampleRUM); }
+    if (!window.hlx?.rum) { initHlxRUM(sampleRUM); }
```

Comment:
We've seen this `?.` break some older browsers, it might make sense to go the longer route.